bibtex-actions.el

```diff
@@ -136,15 +132,22 @@ key associated with each one."
    (let* ((pdf (if (assoc "=has-pdf=" (cdr candidate)) " has:pdf"))
           (note (if (assoc "=has-note=" (cdr candidate)) "has:note"))
           (link (if (assoc "doi" (cdr candidate)) "has:link"))
-          (add (s-trim-right (s-join " " (list pdf note link)))))
+          (citekey (bibtex-completion-get-value "=key=" candidate))
+          (add (s-trim-right (s-join " " (list pdf note link))))
+          ; TODO: add separation between target and suffix
+          (suffix  (bibtex-completion-format-entry candidate (1- (frame-width))
+                        bibtex-completion-display-formats-suffix-internal)))
    (cons
     ;; Here use one string for display, and the other for search.
     ;; The candidate string we use is very long, which is a bit awkward
     ;; when using TAB-completion style multi selection interfaces.
     (propertize
-     (s-append add (car candidate)) 'display (bibtex-completion-format-entry
-     candidate (1- (frame-width))))
-    (cdr (assoc "=key=" candidate))))))
+     (s-append add (car candidate))
+     'display (bibtex-completion-format-entry candidate (1- (frame-width)))
```

Comment:
I don't really understand widths in emacs; is this right?

Reply:
That's what was being used in the original package no? LGTM

Reply:
> That's what was being used in the original package no?

Yes, but I just don't understand it, so wasn't sure if it mattered if we added a suffix with the same. Apparently not.

> LGTM

It's ready to merge, pending Titus merging the bibtex-completion PR.